backend/alembic/versions/0368ff00f62e_create_table.py

```
@@ -12,15 +12,17 @@
 # revision identifiers, used by Alembic.
-revision: str = 'efc2d2b3a635'
-down_revision: Union[str, None] = '16fae346a794'
+revision: str = '0368ff00f62e'
+down_revision: Union[str, None] = None
 branch_labels: Union[str, Sequence[str], None] = None
 depends_on: Union[str, Sequence[str], None] = None
 
 
 def upgrade() -> None:
+    """Upgrade schema."""
     pass
```

Ruff (0.8.2)

Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)